Formula/trimage.rb

```diff
@@ -0,0 +1,25 @@
+class Trimage < Formula
+  desc "Cross-platform tool for optimizing PNG and JPG files"
+  homepage "https://trimage.org"
+  url "https://github.com/Kilian/Trimage/archive/1.0.6.tar.gz"
+  sha256 "60448b5a827691087a1bd016a68f84d8c457fc29179271f310fe5f9fa21415cf"
+
+  depends_on "advancecomp"
+  depends_on "jpegoptim"
+  depends_on "optipng"
+  depends_on "pngcrush"
+  depends_on "pyqt"
+  depends_on "python"
```

Comment:
Please try with `python@3.8`
```suggestion
  depends_on "python@3.8"
```

Reply:
It works with `python`, but fails with `python@3.8` because the `pyqt` dependency that is only installed for `python`.

Reply:
None of the formulas that depend on `pyqt` has been upgraded yet to `python@3.8`.